### PR TITLE
First set of functions needed to complete #61

### PR DESCRIPTION
**Pull request by jspijker**
Added a few functions to download data and some example scripts to test these functions or use them to download a demo database.

The database and download functions are ready and can be merged. We continue #61 to create a UX element which starts the actual download.

### CODE REVIEW

File: funs/download_fun.R [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        loc <- download_locations_knmi(knmistation, time_start, time_end)
+        insert_location_info(station = loc[1, 1],
+                             lat = loc[1, 2],
+                             lon = loc[1, 3],
+                             conn = conn)
+    }
+
+    log_debug("downloading measurements for AQ station")
+    lmlstation <- kitmeta %>%
+        pull(pm10closecode)
```
Comment (elma13) [inline funs/download_fun.R:63]: Here we should also add the pm25close

File: funs/database_fun.R [hunk 2]
```diff
@@ -67,46 +117,80 @@ download_project <- function(project, Tstart, Tend) {
         as_tibble()
 
     log_debug(paste("Got", nrow(stations), "stations for project", project))
-    apply(stations, 1, FUN  = insert_location_info_vectorized, conn = pool)
+    apply(stations, 1, FUN  = insert_location_info_vectorized, conn = conn)
```
Comment (elma13) [inline funs/database_fun.R:121]: Why are you using v1 in line 123, while in 120 you use just the code? What is the difference?

File: funs/database_fun.R [hunk 3]
```diff
@@ -67,46 +117,80 @@ download_project <- function(project, Tstart, Tend) {
...
 
-    info <- get_doc(type = "project", ref = project, conn = pool)
-    kits <- info$sensor_data %>%
-        pull(kit_id)
 
     return(kits)
 }
 
+# 
+# get_stations_from_project <- function(project) {
```
Comment (elma13) [inline funs/database_fun.R:182]: Do we still need this code?

File: scripts/test_functions.R [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+    res <- list(station = s,
+                first_day = first_day,
+                last_day = last_day,
+                time_start = time_range[1],
+                time_end = time_range[2])
+    return(res)
+}
+
+
+station_overview <- function(conn) {
```
Comment (elma13) [inline scripts/test_functions.R:66]: Coool :)

File: scripts/download_demo_database.R [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+
+#  get project info
+download_sensor_meta(project, type = "project")
+
+# get which kits are part of the project
+kits <- get_stations_from_selection(project, type = "project")
+
+for(i in kits) {
+    dl_station(i, time_start, time_end)
```
[the remark this answers is not in the document]
Comment (jspijker) [reply scripts/download_demo_database.R:75]: This looks like you have a broken installation of the ATdatabase package. So it's outside this PR

File: funs/database_fun.R (hunk 2 above)
Comment (jspijker) [reply funs/database_fun.R:121]: Both functions don't return anything, so v1 is obsolete, fixed in https://github.com/rivm-syso/Analyse-Together/pull/75/commits/b5888556f88d07de23fad074c2ccadea42bd60e5

File: funs/database_fun.R (hunk 3 above)
Comment (jspijker) [reply funs/database_fun.R]: No, removed in https://github.com/rivm-syso/Analyse-Together/pull/75/commits/9206d668e39d900653d740d69f79bce4de347999

File: scripts/test_functions.R (hunk 4 above)
Comment (jspijker) [reply scripts/test_functions.R:66]: When we have a few hundredths of stations, we print it out at A0 size, hang it at the office and call it art :-)

File: funs/download_fun.R (hunk 1 above)
Comment (jspijker) [reply funs/download_fun.R]: fixed in https://github.com/rivm-syso/Analyse-Together/pull/75/commits/195842831630de38d48d1e38a6c2920178418d99

**Comment by jspijker** [comment]
Closing this PR without merge, PR is against master (my error) but should be against develop branch.

**Comment by jspijker** [comment]
Reopened, changing base was simpler than I thought ;-)

**Pull request merged**